notebook.ipynb

```diff
@@ -86,43 +86,2150 @@
+       "      <td>5633</td>\n",
+       "      <td>2021-10-10 10:00:00</td>\n",
+       "      <td>2600</td>\n",
+       "      <td>36.758583</td>\n",
+       "      <td>1044.0</td>\n",
+       "      <td>267.0</td>\n",
+       "      <td>178.0</td>\n",
+       "      <td>467.0</td>\n",
+       "      <td>489.0</td>\n",
+       "      <td>89.0</td>\n",
+       "      <td>66.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "        PC4                date  pop_tot      pm2.5  m00_30  m30_60   H1_2  \\\n",
+       "6136   5615 2021-10-30 16:00:00     2724   2.994358   788.0    90.0   23.0   \n",
+       "31664  5645 2021-11-18 08:00:00     1111  13.488200   197.0    12.0   74.0   \n",
+       "8508   5617 2021-10-19 12:00:00     3479   9.960504   607.0   117.0  187.0   \n",
+       "9645   5621 2021-10-11 21:00:00     1290   7.062367   180.0     0.0    0.0   \n",
+       "24130  5633 2021-10-10 10:00:00     2600  36.758583  1044.0   267.0  178.0   \n",
+       "\n",
+       "        H2_4    H4_8  H8_16  H16plus  \n",
+       "6136   495.0   450.0  383.0    495.0  \n",
+       "31664   74.0   210.0  111.0    433.0  \n",
+       "8508   677.0  1681.0  187.0     23.0  \n",
+       "9645     0.0   120.0   30.0    960.0  \n",
+       "24130  467.0   489.0   89.0     66.0  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "zichtop_air_pollution_features = [\n",
+    "    \"PC4\",\n",
+    "    \"date\",\n",
+    "    \"pop_tot\",\n",
+    "    \"pm2.5\",\n",
+    "    \"m00_30\",\n",
+    "    \"m30_60\",\n",
+    "    \"H1_2\",\n",
+    "    \"H2_4\",\n",
+    "    \"H4_8\",\n",
+    "    \"H8_16\",\n",
+    "    \"H16plus\"\n",
+    "]\n",
+    "\n",
+    "df_zichtop_air_pollution = df_zichtop_air_pollution.reindex(zichtop_air_pollution_features, axis=1)\n",
+    "\n",
+    "df_zichtop_air_pollution.sample(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f151668",
+   "metadata": {},
+   "source": [
+    "<h4><font color=mediumvioletred>Rename the date column to date_time </font></h4>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "712399e6",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "df_zichtop_air_pollution.rename(columns={\"date\": \"date_time\"}, inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79cc31f1",
+   "metadata": {},
+   "source": [
+    "<h4><font color=mediumvioletred>Separate date_time column to date and time </font></h4>"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "id": "fa9d31de",
+   "execution_count": 13,
+   "id": "076e5fc0",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "df_zichtop_air_pollution[\"date\"] = df_zichtop_air_pollution[\"date_time\"].dt.date.astype(str)\n",
```

Comment:
is there a reason why you split the date and time in 2 separate columns.

Reply:
Because we wanted to get and visualize the number of people per specific zipcode on specific day.

Reply:
Yes, we wanted to visualize the number of people in zipcode: 5611 on 2021-09-25 per hour
We didn't want to keep showing the date on the x-axis because it is the same date
![image](https://user-images.githubusercontent.com/27498593/203294384-2dc1d382-bb79-471f-ae4b-6b65b7354307.png)